### PR TITLE
updated content for the Notebook Aspect User Guide

### PR DESCRIPTION
**Pull request by ameisner**

### CODE REVIEW

File: docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+**The "notebooks/tutorial-notebooks" directory is read-only:**
+The read-only "notebooks/tutorial-notebooks" directory will *always* contain the most up-to-date versions of the tutorials.
+Notebooks can be edited and executed in this directory, but **changes cannot be saved to this directory**.
+Users wishing to edit, execute, *and save* versions of these notebooks should copy them to a different path in their home directory.
+
+**How to obtain an editable version of a tutorial notebook:**
+The commands below demonstrate how to create a copy of a tutorial notebook in the home directory which can be opened, edited, and saved.
+Step-by-step, the commands below change directory (``cd``) into the home directory (``~``),
+copy the desired tutorial to the (``cp``) into the current directory (``.``),
```
Comment (jeffcarlin) [inline docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst:20]: Maybe change to:

copy (``cp``) the desired tutorial (``notebooks/tutorial-notebooks/<filename>.ipynb``) into the current directory (``.``),

?

File: docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+list (``ls``) the files that are in the current directory to confirm the copy worked,
+and list in long format all attributes in human-readable form (``-lah``) for the copied file.
+The standard output ``-r--r--r--`` indicates that the file is read-only (``r``) by the user, the group, and everyone
+with access to the file (the three ``r``).
+Change the mode (``chmod``) of the file to add user write access (``u+w``), and repeate the
```
Comment (jeffcarlin) [inline docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst:25]: "repeate" --> "repeat"

**Review by jeffcarlin — COMMENTED** [review]
This is great, Aaron! I like the overall organization, and it seems straightforward to navigate. I added a few small comments in the git repo, plus the following comments/questions below:

In "Intro to RSP Notebook Aspect": What is the purpose of [nb.lsst.io](http://nb.lsst.io/), and/or its relationship to this documentation site?

In "How to use a Jupyter NB": I feel like there could be more description of typical interactions. But maybe just a link to basic documentation about interacting with Jupyter notebooks could be enough. (the link that’s a “See also” sidebar in the “Create a new notebook with the LSST kernel” section might be sufficient — seems OK to repeat it in both places?)

In "How to use a Jupyter NB": Should we add a link from here to the “create a new notebook with the LSST kernel” section?  Otherwise it feels like all people can do is run premade notebooks.

In "How to use the Tutorial Notebooks": Should we remind people that Jupyter NB cells can be executed out of order, and that this can have benefits and drawbacks? (And that an easy way to get yourself out of a jam created by executing things out of order is to restart the kernel?)

In "Using the LSST Science Pipelines in the terminal": I’m unsure about this, but I feel like some more examples of things one might do from the terminal would be useful. This is tricky, though, since users only have access to the CPU/memory resources of the kernel. So, for example, I would NOT point them to the “Getting started” guide on [pipelines.lsst.io](http://pipelines.lsst.io/), even though that seems like a logical first step. Maybe just add a link to “some examples” at https://dp0-2.lsst.io/tutorials-examples/index.html#command-line-tutorials? 

Should we add a note somewhere about how to monitor container memory usage (i.e., the "current resource usage" displayed at the bottom of the page, and what happens/how to recover when you exceed your container limit?

File: docs/guides/notebooks/faq/index.rst [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+because in that situation the Butler is used for image access and the result is a user-generated catalog persisted by and accessible with Butler.
+Furthermore, a few data products are only available via the Butler, such as survey property maps, raw images, and source footprints.
+
+.. _NB-Intro-Use-A-NB-faq-questions:
+
+How do I ask questions about Notebooks?
+---------------------------------------
+
+`Get support at any time via the `Rubin Observatory Community Forum <https://community.lsst.org/>`_ or via GitHub Issues.
+Another option is to attend the biweekly Rubin Science Assemblies which will feature live tutorials and question-and-answer time with Rubin Observatory staff.
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/faq/index.rst:157]: Yeah, I agree it feels quite reasonable to link to the Rubin Science Assemblies schedule here. But I understood it to be a strict requirement not to include any mention of DP0, or even any hyperlinks that include "dp0" in them, hence why there's currently not a link (the idea is that content within rsp.lsst.io should be independent of any particular data release, with specifics about a particular data release confined to that data release's documentation website).

In any case, I'll remove "biweekly" here.

[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/faq/index.rst]: "biweekly" removed in [4d7758b](https://github.com/lsst/rsp_lsst_io/commit/4d7758bc365757723a5148506442354a469f8a8d) 👍

File: docs/guides/notebooks/faq/index.rst [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+The Butler is also used when doing any kind of image reprocessing (e.g., re-doing source detection),
+because in that situation the Butler is used for image access and the result is a user-generated catalog persisted by and accessible with Butler.
+Furthermore, a few data products are only available via the Butler, such as survey property maps, raw images, and source footprints.
+
+.. _NB-Intro-Use-A-NB-faq-questions:
+
+How do I ask questions about Notebooks?
+---------------------------------------
+
+`Get support at any time via the `Rubin Observatory Community Forum <https://community.lsst.org/>`_ or via GitHub Issues.
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/faq/index.rst]: Thanks, good catch! I've fixed this in [a3cd0f9](https://github.com/lsst/rsp_lsst_io/commit/a3cd0f9edfd3a7716928a0fd5ec5137144931b48)

File: docs/guides/notebooks/faq/index.rst [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+--------------------------
+
+Yes, the RSP Notebook Aspect will only have python environments.
+
+To access data from the Notebook Aspect, users will need to use Python commands and code.
+Much of the LSST Science Pipelines code is in Python, and the Rubin/LSST tutorial notebooks use Python as well.
+These tutorials contain executable examples of the commands required to access and analyze data.
+All Rubin/LSST users should feel free to copy and paste from the provided tutorials.
+
+Anyone new to Python and looking to learn more might benefit from this `Python for Beginners <https://www.python.org/about/gettingstarted>`_ website (which includes links to tutorial in a variety of languages),
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/faq/index.rst]: Thanks, good catch -- fixed in [9488890](https://github.com/lsst/rsp_lsst_io/commit/9488890ffc555ddf5674d050299610e6a458716e)

File: docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst (hunk 2 above)
Comment (ameisner) [reply docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst]: Good catch, fixed in [d7b6135](https://github.com/lsst/rsp_lsst_io/commit/d7b6135d513e384b89f2195aec734fa8cb3d14f8), thanks

File: docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst [hunk 6]
```diff
@@ -0,0 +1,43 @@
+##################
+Tutorial Notebooks
+##################
+
+.. _NB-Intro-Use-Tutorial-NBs:
+
+How to use the Tutorial Notebooks
+=================================
+
+The best way to learn how to use a Jupyter Notebook is to open the first of the Rubin/LSST tutorial notebooks which are provided in each user's home directory.
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst:10]: It does feel like it'd make sense to mention a specific notebook, but my instructions were to not make any specific reference to DP0, and even avoid including "dp0" within any hyperlink. That being said, I can still be more explicit about the location ~/notebooks/tutorial-notebooks, so I'll edit accordingly.

[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst]: Implemented a change in response to this in [b790fe9](https://github.com/lsst/rsp_lsst_io/commit/b790fe92fe5c04aa0ec48673d6ec692343346023)

File: docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst (hunk 1 above)
Comment (ameisner) [reply docs/guides/notebooks/jupyter-notebooks/tutorial-notebooks.rst]: Yes, good call, I've implemented this in [0023f47](https://github.com/lsst/rsp_lsst_io/commit/0023f47b634c96ef2f0b9c37011ecd66a5e21d55).

File: docs/guides/notebooks/starting-and-stopping/index.rst [hunk 7]
```diff
@@ -1,11 +1,66 @@
...
+      :alt: This image is a screenshot of the main work area of the Notebook Aspect, as it appears when a user starts a new server. Across the top is the main menu, with options such as file, edit, view, run, kernel, rubin, tabs, settings, and help. The left sidebar offers options to browse the file system, open files, and upload data. At right, in the main work area, one tab is open. It is the launcher tab, which offers options to open a new notebook, coding console, terminal, text file, markdown file, python file, or help file.
+      :width: 400
+      :name: RSP_NB_launcher_options
+
+      The JupyterLab landing page from which several tools can be launched and the file system can be browsed (left sidebar).
+
+In the very left-most vertical sidebar of icons, the top icon is a file folder, and that is the default view.
+The left sidebar lists folders in the user's home directory (e.g., DATA, WORK, and notebooks).
+Launching a terminal (the default is a linux bash terminal) and using the command "ls" will return the same list.
+Navigate the file system and open files by double-clicking on folders and files in the left sidebar.
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/starting-and-stopping/index.rst:53]: Good idea, thanks, done in [b2aa64a](https://github.com/lsst/rsp_lsst_io/commit/b2aa64a870b3ba08d947645ef4aa14013adbbe4b).

File: docs/guides/notebooks/starting-and-stopping/index.rst [hunk 8]
```diff
@@ -1,11 +1,66 @@
...
+
+Although the file browser is a handy way to navigate the user home space, it does not allow navagating to e.g., the shared data space.
+One way to make other spaces available in the file browser is to create a `symbolic link <https://en.m.wikipedia.org/wiki/Symbolic_link>`_ pointing to the desired resource using the Terminal, with this symbolic link placed somewhere in the home directory.
+
+Jupyter Notebooks can be identified by their file extension ".ipynb".
+All users will find a set of tutorial notebooks provided in the "notebooks/tutorial-notebooks/" directory.
+
+**Safely Log Out of JupyterLab:**
+Use the "File" menu in the top menu bar.
+To safely shut down a Notebook, choose "Close and Shutdown Notebook".
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/starting-and-stopping/index.rst:64]: thanks, added a comment about this in [87d24d2](https://github.com/lsst/rsp_lsst_io/commit/87d24d2bf6b11bb9ab762708f60795f8fd9151e8)

File: docs/guides/notebooks/jupyter-notebooks/index.rst [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+There is a `user guide for markdown cells in the Jupyter Notebooks documentation <https://jupyter-notebook.readthedocs.io/en/stable/examples/Notebook/Working%20With%20Markdown%20Cells.html>`_.
+
+  .. figure:: images/notebook.png
+      :name: notebook_aspect
+      :alt: This image is a screenshot of the tutorial notebook entitled Introduction to Jupyter Notebooks for Data Preview 0.2. The notebook has been scrolled down to Section 3.3, which contains both markdown text and code cells which have been executed. The last code cell has produced a greyscale image of a rich galaxy cluster. Across the top of the notebook there is a menu bar of actions for users. Actions include save notebook, set cell type, and insert, cut, copy, paste, run, or interrupt cells.
+
+      A screenshot from a Rubin/LSST tutorial notebook, showing the panel of the Notebook Aspect where multiple interface tabs can be open at once. In this case, the first tab is a command-line terminal, the second is the Launcher interface, and the third (which is currently selected) is an executed version of a tutorial notebook. Multiple notebooks can be opened in separate tabs.
```
[the remark this answers is not in the document]
Comment (ameisner) [reply docs/guides/notebooks/jupyter-notebooks/index.rst]: Good catch, I updated the screenshot image and ensured that the caption and alt text now match the new screenshot.

**Pull request merged**